reactDashboard/src/components/InfoBar/InfoBar.css

```diff
@@ -1,16 +1,28 @@
 .info-bar {
   background-color: #c1def1;
-  padding: 10px 40px 10px 20px;
+  padding: 10px 20px;
+}
+
+/* reverted changes to meet accessibility requirements */
+.darkmode .info-bar {
+   opacity: 0.75;
```

Comment:
I don't get that - if the end rendered colour is the same, why does using opacity fool the accessibility check - seems like a cheat

Reply:
probably something to do with the transparency aspect of opacity - I don't believe you can just make it a different color - it isn't fooling the accessibility check - it genuinely is more readable/better contrast with my settings

Reply:
That means that the background colour just needs adjusting. If you have a white background and you put a 50% opaque black box on top of it, the colour of the box just appears grey. Might as well just make the background colour of the box grey - the end result is identical. The only reason to use transparency is if you want to see content underneath, which you don't in this case.

Reply:
I lost!